src/clj/game/cards/operations.clj

```diff
@@ -291,6 +291,23 @@
                               :effect (effect (trash target))}
                             card nil)))}}
 
+   "Enhanced Login Protocol"
+   (let [remove-effect (req (when (:elp-click-run card)
+                              (click-run-cost-bonus state side [:click -1])
+                              (update! state side (dissoc card :elp-click-run))))
+         remove-ability {:req (req (and
+                                     (:elp-click-run card)
+                                     (:elp-fired card)))
+                         :effect remove-effect}]
+     {:events {:runner-turn-begins {:msg "add an additional cost of 1 [Click] to make the first run not through a card ability this turn"
+                                    :effect (effect (update! (assoc card :elp-click-run true))
+                                              (click-run-cost-bonus [:click 1]))}
```

Comment:
Oh wait. This is actually more subtle than that, because it's activated by the first click run each turn, which could have happened before playing it.